chargeio/infrastructure/CustomDateTimeConverter.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using Newtonsoft.Json.Converters;
+
+namespace chargeio
+{
+    class CustomDateTimeConverter : IsoDateTimeConverter
```

Comment:
fixed in https://github.com/affinipay/chargeio-net/pull/6/commits/1a7f482ebf08cf49cb5e5d24750eb94e656de02a